modules/_node-scripts/esbuild/runEsbuild.mjs

```diff
@@ -0,0 +1,34 @@
+import esbuild from 'esbuild';
+import fs from 'fs/promises';
+import path from 'path';
+
+export default async function runEsbuild(config, configName) {
+	return Promise.all([
+		doRunEsbuild(config, configName),
+		writeDebugEsbuildConfig(config, configName),
+	]);
+}
+
+async function doRunEsbuild(config, configName) {
+	const start = performance.now();
+
+	try {
+		await esbuild.build(config);
+	}
+	catch(error) {
+		throw new Error(`Esbuild command failed: ${error}`);
+	}
+
+	const lapse = performance.now() - start;
+
+	console.log(`Esbuild for ${configName} took: ${(lapse/1000).toFixed(3)} s`);
+}
+
+async function writeDebugEsbuildConfig(config, configName) {
+	const configFilePath = path.join(
+		'build', 'node-build', `${configName}.esbuild.config.json`
```

Comment:
WIll this get bundled in the jar?

Reply:
Nope. It's only for build debugging purposes.